Prioritize/README.md

```diff
@@ -46,7 +46,7 @@ on:
 ## Prioritize Troubleshooting
 * Add ```-viaDebug true``` at the end of the Unified Agent command
 * Publish the following folders using your pipeline publish tool, [GitHub Prioritize Log Publish example](#GitHub-Prioritize-Log-Publish)
-  * /tmp/whitesource*
+  * /tmp/Mend*
```

Comment:
Same as before, is this folder name supported? Is it the (still) hard-coded ./whitesource folder the UA creates, or is this an arbitrary name in this context?